relation_engine_server/utils/json_validation.py

```diff
@@ -165,7 +163,8 @@ def resolve_remote(self, uri):
             result = requests.get(uri).json()
         else:
             # Otherwise, pass off to urllib and assume utf-8
-            with urlopen(uri) as url:
+            # TODO https://github.com/kbase/relation_engine/issues/156
```

Comment:
Seems like this is for the remote specs loading that we don't even use? Maybe? If so, YAGNI for now

Reply:
Can we confirm we don't use remote specs and we're not going to? If so we could just remove this class (and maybe more stuff)